Avalanche Stats (#312)
ava-labs/avalanche-docs: build/tools/avalanche-stats.md

# Avalanche Stats

## Introduction

> We created a [stats][https://stats.avax.network/] website that you can learn a lot of things about avalanche's current status.

Reviewer: ```suggestion
We have created a [stats][https://stats.avax.network/] website from where you can learn a lot about avalanche's current stats.
```

> ## What is in there

Reviewer: ```suggestion
## What are included?
```

> We have 3 main concepts in our stats: Staking, Validator Health Check and C-Chain Activity.

Reviewer: ```suggestion
We have 3 main areas in our stats: Staking, Validator Health Check and C-Chain Activity.
```

## Staking

> You can access this part from [this][https://stats.avax.network/dashboard/staking/] link. And you can see:

Reviewer: ```suggestion
You can access this part from [this][https://stats.avax.network/dashboard/staking/] link from which you will get following info:
```

Amount of staked by your node (in AVAX).

### Delegation Fee (%)

You can learn more about this from [here][https://docs.avax.network/learn/platform-overview/staking#delegator-rewards]

### Delegators

Shows how many delegates the node has.

### Delegations

Shows the amount delegates have staked on your node.

### Benched, Offline and Inaccessible (last 48 hours)

Those are graphichs that shows your node's status for benched, offline and inaccessible status.

## C-Chain Activity

You can see our C-Chain stats from [here][https://stats.avax.network/dashboard/c-chain-activity/].

### Max TPS Observed (All-Time, last 7 days, last 30 minutes)

Don't think this is the limit for Avalanche. We got 11000+ tps according to the tests we made on amazon servers located in different points around the world. 

### Block Size (Last 2000)

You can see the last 2000 block's size (KB) in this graphic.

### Tx Count (Last 2000)

You can see how many transactions were in the last 2000 block in this graphic.

### Minumum Gas Price (Last 2000)

Minumum gas price for the last 2000 block.

### Minumum Gas Price (Last 8 hours)

Minumum gas price for the last 8 hours.

### Daily Fees Burned

> Shows how much gas is burned in the last day.

Reviewer: ```suggestion
Shows how much gas is burned in every day.
```

### Daily Gas Used

> Shows how much gas is used in the last day.

Reviewer: ```suggestion
Shows how much gas is used in every day.
```

### Daily Tx Count

> Shows how many transaction were made in the last day.

Reviewer: ```suggestion
Shows how many transaction were made in every day.
```
